src/codegen/pythoncg.cpp

```diff
@@ -24,6 +24,9 @@
 // Python code generation writen by
 //   Michal Bližňak - michal.bliznak@gmail.com
 //
+//   Jacobo Costas Costas
```

Comment:
Don't add credits for small fixes or additions. So far these credits have only been used on large additions like support for a new generator language.